projects/angular/src/accordion/accordion-panel.html

```diff
@@ -1,6 +1,10 @@
 <ng-container *ngIf="panel | async; let panel">
   <div [ngClass]="getPanelStateClasses(panel)">
-    <div class="clr-accordion-header">
+    <div
+      class="clr-accordion-header"
+      [attr.role]="headingEnabled ? 'heading' : null"
+      [attr.aria-level]="headingEnabled ? headingLevel : null"
```

Comment:
```suggestion
      [attr.role]="headingEnabled || explicitHeadingLevel ? 'heading' : null"
      [attr.aria-level]="headingEnabled || explicitHeadingLevel ? headingLevel : null"
```

I don't think the current implementation is quite right. If the user sets an explicit heading level, it should work without the feature flag being enabled (setting the heading level implicitly means the user wants the accordion header to be `role="heading`).

Reply:
It is right.

If there is not flag: `role="heading"` will be always applyed. 
If there is not flag: ` [attr.aria-level]="headingLevel"` will be applyed. 

`explicitHeadingLevel` is only for that `explicit`, `headingLevel` will alway have value.

Reply:
Logic updated in ac47ae2

Reply:
Thanks!